<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BackgroundCfger</class>
 <widget class="QGroupBox" name="BackgroundCfger">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>199</width>
    <height>230</height>
   </rect>
  </property>
  <property name="title">
   <string>Background</string>
  </property>
  <property name="alignment">
   <set>Qt::AlignCenter</set>
  </property>
  <property name="flat">
   <bool>true</bool>
  </property>
  <property name="checkable">
   <bool>true</bool>
  </property>
  <property name="checked">
   <bool>false</bool>
  </property>
  <widget class="QComboBox" name="comboBoxStyle">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="iconSize">
    <size>
     <width>64</width>
     <height>8</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="labelDirection">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>40</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Speed:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxSpeed">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>70</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="maximum">
    <number>20</number>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonClassic">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>110</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Classic</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonFreestyle">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>150</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Freestyle</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonSlide">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>190</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Slide</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>40</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Style:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
